src/doc/doc_phases/doc_html.nit

```diff
@@ -532,267 +464,142 @@ redef class DocComposite
 	end
 end
 
-redef class DocRoot
-
-	# Start the rendering from root.
-	#
-	# FIXME needed to maintain TplSection compatibility.
-	fun start_rendering(v: RenderHTMLPhase, doc: DocModel, page: MEntityPage): TplSection do
-		var section = new TplSection("top")
-		var mentity = page.mentity
-		section.title = mentity.html_name
-		section.subtitle = mentity.html_declaration
-		# FIXME ugly hack to avoid diff
+# FIXME hideous hacks to avoid diff
```

Comment:
So, we left the ugly hack to find a collection of hideous hacks ?
Sounds like proper improvement from my point of view :)